src/uwtools/tests/config/test_validator.py

```diff
@@ -135,13 +136,123 @@ def test_validate_workflow_tags_pass(vals):
     fn, validity = vals
     with resources.as_file(resources.files("uwtools.resources")) as resc:
         schema_file = resc / "rocoto.jsonschema"
-    config_file = support.fixture_path(fn)
+    config_file = fixture_path(fn)
     assert (
         validator.validate_yaml(schema_file=schema_file, config_file=config_file, check_path=False)
         is validity
     )
 
```

Comment:
I think with the addition of the `@fixture` tests that Paul fleshed out we should be able to completely remove the `test_validate_workflow_tags_pass` along with their respective`hello_workflow_tags.yaml` and `hello_workflow_invalid.yaml` files that are referenced within the test case. 

Eeventually when we merge, `hello_workflow.yaml` should remain since past work makes use of this file.